firmware/application/apps/ui_btle_rx.cpp

```diff
@@ -70,7 +70,7 @@ BTLERxView::BTLERxView(NavigationView& nav)
     };
 
     // Auto-configure modem for LCR RX (will be removed later)
-    baseband::set_btle(persistent_memory::modem_baudrate(), 8, 0, false);
+    baseband::set_btle(channel_number);
```

Comment:
Re: settings, you could also persist the channel if you want.
Doesn't have to be this PR either -- perfectly fine as is.